lttnganalyses/core/period.py

```diff
@@ -480,7 +481,7 @@ def _not_expr_matches(self, expr):
         return not self._expr_matches(expr.expr)
 
     def _glob_eq_expr_matches(self, expr):
-        compfn = lambda lh, rh: bool(expr.regex.match(lh))
+        def compfn(lh, rh): return bool(expr.regex.match(lh))
```

Comment:
@eepp can you confirm this is a good fix for "E731 do not assign a lambda expression, use a def" ?